core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -33,7 +34,7 @@ import org.apache.kafka.common.protocol.types.Struct
 import org.apache.kafka.common.resource.ResourceNameType
 import org.apache.kafka.common.record._
 import org.apache.kafka.common.requests.CreateAclsRequest.AclCreation
-import org.apache.kafka.common.requests.{Resource => RResource, ResourceType => RResourceType, _}
+import org.apache.kafka.common.requests._
```

Comment:
Should we really be using star imports?

Reply:
In my defence, I've only removed imports here, not added them.